docs/data/data-grid/columns/ColumnSpanningNumber.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { DataGrid } from '@mui/x-data-grid';
+
+const otherProps = {
```

Comment:
We usually use `other`.

```suggestion
const other = {
```